### PR TITLE
Add watchOS Support to leveldb-library

### PR DESCRIPTION
**Pull request by ryanwilson**
This is needed for Firebase Database watchOS support.

### CODE REVIEW

File: leveldb-library.podspec [hunk 1]
```diff
@@ -11,6 +11,7 @@ Pod::Spec.new do |s|
   s.ios.deployment_target = '5.0'
   s.osx.deployment_target = '10.7'
   s.tvos.deployment_target = '10.0'
+  s.watchos.deployment_target = '6.0'
 
   s.source       =  {
```
Comment (paulb777) [inline leveldb-library.podspec:16]: The line below will need to be hardcoded to 1.22

Comment (paulb777) [reply leveldb-library.podspec:16]: with a comment

Comment (ryanwilson) [reply leveldb-library.podspec:17]: Thanks!

**Review by paulb777 — APPROVED** [review]

**Comment by paulb777** [comment]
Please publish to SpecsStaging and make sure there's a clean nightly before publishing

**Comment by ryanwilson** [comment]
Thanks, staged now: https://github.com/firebase/SpecsStaging/commit/527f672533295d89b2399d8073772847f9abba61

**Comment by paulb777** [comment]
Published at https://github.com/CocoaPods/Specs/blob/master/Specs/a/d/a/leveldb-library/1.22.1/leveldb-library.podspec.json

**Pull request merged**